src/docfx/lib/JsonUtility.cs

```diff
@@ -324,6 +324,28 @@ public static Range ToRange(IJsonLineInfo lineInfo)
             return lineInfo != null && lineInfo.HasLineInfo() ? new Range(lineInfo.LineNumber, lineInfo.LinePosition) : default;
         }
 
+        public static List<Error> IncludeAll(List<Range> ranges, Error error)
```

Comment:
I  believe we have a WithRange() in Error.cs, we can put this method there named WithRanges()

Reply:
Hold on this PR to wait for #4298 finished